### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by jglick**
For JEP-200 testing it is useful to have a modern POM.

@reviewbybees

### CODE REVIEW

File: src/main/java/com/tikal/jenkins/plugins/multijob/MultiJobProject.java [hunk 1]
```diff
@@ -158,4 +158,23 @@ protected void submit(StaplerRequest req, StaplerResponse rsp) throws IOExceptio
             setResumeEnvVars(resumeEnvVars);
         }
     }
+
+    // TODO as in https://github.com/jenkinsci/maven-plugin/pull/109:
```
Comment (jglick) [inline src/main/java/com/tikal/jenkins/plugins/multijob/MultiJobProject.java]: https://github.com/jenkinsci/maven-plugin/pull/109, for the backlinks

File: src/main/java/com/tikal/jenkins/plugins/multijob/MultiJobBuild.java [hunk 2]
```diff
@@ -93,7 +93,7 @@ public String getBuildParams(SubBuild subBuild) {
                 } catch (Exception e) {
                     continue;
                 }
-                String value = stringParameter.value;
+                String value = (String) stringParameter.getValue();
```
Comment (oleg-nenashev) [inline src/main/java/com/tikal/jenkins/plugins/multijob/MultiJobBuild.java]: Hmm, would be useful to patch the core if it's still not fixed. Will cause FindBugs warnings, but does not matter so far

**Comment by oleg-nenashev** [comment]
@reviewbybees done

File: src/main/java/com/tikal/jenkins/plugins/multijob/MultiJobBuild.java (hunk 2 above)
Comment (jglick) [reply src/main/java/com/tikal/jenkins/plugins/multijob/MultiJobBuild.java]: The method is declared to return `Object` even though the actual value is a `String`. Could be improved in core but hardly seems a priority.

**Pull request merged**